help/C/Help_ch_Transactions.xml

```diff
@@ -43,8 +43,9 @@ Translators:
     </listitem>
 
     <listitem>
-      <para>Importing transactions either from a <acronym>QIF</acronym>/<acronym>OFX</acronym> file or via
-      "<acronym>OFX</acronym> on-line banking".</para>
+      <para>Importing transactions either from an <acronym>OFX</acronym>,
+        <acronym>QFX</acronym>, <acronym>QIF</acronym> or <acronym>CSV</acronym>
+        file or via "<acronym>OFX</acronym> on-line banking".</para>
     </listitem>
```

Comment:
I have changed it to say:
"Importing transactions either from a file or via on-line banking." as you suggest.